src/openassetio-python/tests/package/hostApi/test_manager.py

```diff
@@ -258,6 +259,397 @@ def test_when_entity_ref_prefix_type_invalid_then_debug_log_printed(
 )
 
 
+class BatchFirstMethodTest:
```

Comment:
[Question] What does the "first" mean here? Confused by the name

Reply:
Oh i get it, "BatchFirstMethods" meaning, test methods that fit our openassetio convenience pattern, otherwise known as the batch first methods :+1: